app/questionnaire/questionnaire_schema.py

```diff
@@ -935,3 +975,70 @@ def get_item_anchor(self, section_id: str, list_name: str) -> Optional[str]:
             for item in summary.get("items", []):
                 if item["for_list"] == list_name and item.get("item_anchor_answer_id"):
                     return f"#{str(item['item_anchor_answer_id'])}"
+
+    def get_block_ids_for_calculated_summary_dependencies(
```

Comment:
This doesn't seem right with 2 circular dependencies, let's move it to a new file called `dependencies.py` in `app/questionnaire`?

Reply:
Moved this now 👍